endpoint.go

```diff
@@ -500,8 +500,10 @@ func (ep *endpoint) sbJoin(sb *sandbox, options ...EndpointOption) error {
 		return err
 	}
 
-	if e := ep.addToCluster(); e != nil {
-		log.Errorf("Could not update state for endpoint %s into cluster: %v", ep.Name(), e)
+	if sb.ingress {
```

Comment:
@mavenugo: for managed containers engine will call the Enable/Disable service API. Since the ingress sandbox is created specially its handled explicitly here. Alternative would be to call EnableService [here] (https://github.com/docker/docker/blob/master/daemon/network.go#L176)  after the `ingress-endpoint` join to the `ingress' sandbox.